cmd/ndm_daemonset/probe/mountprobe.go

```diff
@@ -31,11 +35,15 @@ type mountProbe struct {
 	// create, update, delete, deactivate the disk resources or list the probes already registered.
 	Controller      *controller.Controller
 	MountIdentifier *mount.Identifier
+	epoll           *epoll.Epoll
+	destination     chan controller.EventMessage
+	mountsFileName  string
 }
 
 const (
 	mountProbePriority = 4
 	mountConfigKey     = "mount-probe"
+	defaultMountsFile  = "/proc/1/mounts"
```

Comment:
Can use `mount.HostMountsFilePath` here, since for checking the mount points on the host machine we need the procfs of the host. `/proc/1/mounts` inside the container will only give the mounts inside the container.